my_game.py

```diff
@@ -170,6 +187,19 @@ def on_draw(self):
             bold=True,
         )
 
+
+        # Draw the the spritelists in the tilemap
```

Comment:
Something is wrong here. This is not the way we draw the spritelists in the tilemap: https://github.com/tobiasbp/python-arcade-dungeon/blob/main/my_game.py#L154

Reply:
I believe that that is from an older version af the file or something...
because the current origin/enemy-parent-class my_game is identical with main on that point, look:
https://github.com/tobiasbp/python-arcade-dungeon/blob/enemy_parent_class/my_game.py#L164

Reply:
Then remove 191-193

Reply:
oh that's right